apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -127,20 +125,101 @@ function LandingPage({
     getEnrollSucessWorkshopName()
   );
   const [currentTab, setCurrentTab] = useState(availableTabs[0].value);
+  const [workshopsAsFacilitator, setWorkshopsAsFacilitator] = useState([]);
+  const [workshopsAsOrganizer, setWorkshopsAsOrganizer] = useState([]);
+  const [workshopsAsProgramManager, setWorkshopsAsProgramManager] = useState(
+    []
+  );
+
   const headerContainerStyles =
     availableTabs.length > 1
       ? style.headerWithTabsContainer
       : style.headerWithoutTabsContainer;
-
   const joinedPlSectionsStyling =
     joinedPlSections?.length > 0 ? '' : style.joinedPlSectionsWithNoSections;
 
-  // Load PL section info into redux
+  // Load PL section into redux and fetch applicable workshop info
   const dispatch = useDispatch();
   useEffect(() => {
     dispatch(asyncLoadSectionData());
     dispatch(asyncLoadCoteacherInvite());
-  }, [dispatch]);
+
+    if (userPermissions.includes('facilitator')) {
+      const fetchFacilitatorData = async () => {
+        try {
+          const response = await fetch(
+            '/dashboardapi/v1/pd/workshops_as_facilitator_for_pl_page',
+            {
+              method: 'GET',
+              headers: {
+                'Content-Type': 'application/json',
+                'X-CSRF-Token': await getAuthenticityToken(),
+              },
+            }
+          );
+
+          if (response.ok) {
+            const jsonData = await response.json();
+            setWorkshopsAsFacilitator(jsonData.workshops_as_facilitator);
+          }
+        } catch (error) {
+          console.error('Error fetching facilitator data:', error);
+        }
+      };
+
+      fetchFacilitatorData();
+    }
+
+    if (userPermissions.includes('workshop_organizer')) {
+      const fetchWorkshopOrganizerData = async () => {
+        try {
+          const response = await fetch(
+            '/dashboardapi/v1/pd/workshops_as_organizer_for_pl_page',
+            {
+              method: 'GET',
+              headers: {
+                'Content-Type': 'application/json',
```

Comment:
do you need the auth token here and on line 207 as well?

Reply:
Ah yes you're right thank you!